src/app/projects/states/dashboard/directives/progress-dashboard/progress-dashboard.coffee

```diff
@@ -1,19 +1,17 @@
 angular.module('doubtfire.projects.states.dashboard.directives.progress-dashboard', [])
-#
-# Summary dashboard showing some graphs and way to change the
-# current target grade
-#
-.directive('progressDashboard', ->
+.directive 'progressDashboard', ->
```

Comment:
Can We use Angular Components instead of CoffeeScript for new items, we are moving away from this.